### ISSUE TITLE
KB2919422 update is missing to upgrade Windows Embedded 8.1 to Update 1

### ISSUE THREAD

**Issue by tretdm** [LegacyUpdate/LegacyUpdate#284]
**This update must be installed before KB2919355.** This is the likely reason why installing Legacy Update on Windows 8.1 causes a boot loop.

**Comment by stdin82** [comment]
KB2919422 = lower version of servicing stack update KB3021910

**Comment by kirb** [comment]
Yep, it's not necessary for a working install of 8.1 Update 1. The set of updates for 8.1 hasn't changed since I added it I believe. The root problem is likely rather what I discuss [here](https://github.com/LegacyUpdate/LegacyUpdate/issues/285#issuecomment-2445591600), which is also affecting 7 (and probably Vista).

**Comment by tretdm** [comment]
Apparently, this only affected the Embedded editions of Windows 8.1, which the Legacy Update was not designed for.

**Comment by kirb** [comment]
All editions are intended to be supported, but I might have broken something with Embedded 8.1 in 1.10. I'll give it a test.

**Comment by kirb** [comment]
Can't reproduce any issues with LU 1.10 on a clean install of Embedded 8.1 Industry Pro x64. I left it at defaults (everything selected in the setup), and it successfully installed Update 1 and did two reboots.

**Comment by tretdm** [comment]
And I will also add that this happens specifically with the Russian-language version of Windows Embedded 8.1.

**Comment by kirb** [comment]
Since you mention it happening with a Russian install, this would be issue #285, so closing in favor of that. Thanks for reporting, wasn’t aware this is still an issue on 8.1 in addition to 7.
